content/articles/2019-08-googledrive-1-0-0.Rmarkdown

```diff
@@ -0,0 +1,115 @@
+---
+title: googledrive v1.0.0
+slug: googledrive-1-0-0
+author: Jenny Bryan
+description: >
+  googledrive v1.0.0 is on CRAN.
+date: '2019-08-21'
+categories: [package]
+tags:
+  - googledrive
+photo:
+  url: https://unsplash.com/photos/8EzNkvLQosk
+  author: Maarten van den Heuvel
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+## Introduction
+
+We're jazzed to announce the release of [googledrive](https://googledrive.tidyverse.org) v1.0.0.
+
+googledrive wraps the [Drive REST API v3](https://developers.google.com/drive/). The most common file operations are implemented in high-level functions designed for ease of use. You can find, list, create, trash, delete, rename, move, copy, browse, download, share and publish Drive files, including those on Team Drives.
+
+Install googledrive with:
+
+```{r eval = FALSE}
+install.packages("googledrive")
+```
+
+The release of version 1.0.0 marks two events:
+
+  * The overall design of googledrive has survived ~2 years on CRAN, with very little need for change. The interface and feature set are fairly stable. googledrive facilitated around 7 million requests to the Drive API in the past month. But also ...
+  * There are changes in the auth interface that are not backwards compatible.
+  
+There is also new functionality that makes it less likely you'll create multiple files with the same name, without actually meaning to.
+
+## Auth from gargle
+
+googledrive's auth functionality now comes from the [gargle package](https://gargle.r-lib.org), which provides R infrastructure to work with Google APIs, in general. We've just blogged about gargle's initial release, so check out that post for more details.
+
+We're adopting gargle for auth in several other packages, such as [bigrquery](https://bigrquery.r-dbi.org) (>= v1.2.0), [gmailr](https://gmailr.r-lib.org)  (>= v1.0.0 *coming soon to CRAN*), and [googlesheets4](https://googlesheets4.tidyverse.org) (*currently GitHub-only, successor of googlesheets). This makes new token flows available in these packages, such as [Application Default Credentials](https://www.jhanley.com/google-cloud-application-default-credentials/), and makes auth less idiosyncratic.
```

Comment:
missing closing `*` in `currently GitHub-only`.

Reply:
Fixed (by removing the first `*`.)